django_project/frontend/src/containers/MainPage/Data/index.tsx

```diff
@@ -39,21 +39,21 @@ const DataList = () => {
     const [tableData, setTableData] = useState<any>()
     const [activityTableGrid, setActivityTable] = useState<any>()
     const [userRole, setUserRole] = useState<string>('')
-    const dataset = (userRole ==="Organisation member" || userRole ==="Organisation manager") ?  data.filter(item => !item?.Activity_report)?.flatMap((each) => Object.keys(each)) : data.flatMap((each) => Object.keys(each));
+    const dataset = (userRole ==="Organisation member" || userRole ==="Organisation manager" || userRole === "National data scientist" || userRole === "Regional data scientist") ?  data.filter(item => !item?.Activity_report)?.flatMap((each) => Object.keys(each)) : data.flatMap((each) => Object.keys(each));
```

Comment:
Please create a function here to check user_role.

Reply:
Okay